e2e-tests/cypress/e2e/Reporting.cy.ts

```diff
@@ -0,0 +1,63 @@
+import { Login } from '../support/page-objects/Login';
+import { Loader } from '../support/page-objects/Loader';
+import { esApiClient } from '../support/helpers/EsApiClient';
+import { Home } from '../support/page-objects/Home';
+import { kbnApiClient } from '../support/helpers/KbnApiClient';
+import { getKibanaVersion, userCredentials } from '../support/helpers';
+import { Discover } from '../support/page-objects/Discover';
+import { Reporting } from '../support/page-objects/Reporting';
+import { esApiAdvancedClient } from '../support/helpers/EsApiAdvancedClient';
+import { kbnApiAdvancedClient } from '../support/helpers/KbnApiAdvancedClient';
+import * as semver from 'semver';
+import { IndexLifecyclesPolicies } from '../support/page-objects/IndexLifecyclesPolicies';
+
+if (semver.gte(getKibanaVersion(), '8.15.0')) {
+  describe('Reporting', () => {
+    const oldFormatReportingIndex = '.reporting.kibana_admins_group-2025-02-02';
+    const newFormatReportingName = 'new format reporting index doc';
+    let oldFormatReportingName: string;
+
+    beforeEach(() => {
+      cy.fixture('old_format_reporting_doc.json').then(oldFormatReportingDoc => {
+        oldFormatReportingName = oldFormatReportingDoc.payload.title;
+        esApiClient.addDocument(oldFormatReportingIndex, oldFormatReportingDoc.id, oldFormatReportingDoc);
+        esApiClient.attachLifecyclePolicy(oldFormatReportingIndex, 'kibana-reporting');
+      });
+
+      cy.visit(Cypress.config().baseUrl);
+      cy.on('url:changed', () => {
+        sessionStorage.setItem('ror:ignoreTrialInfo', 'true');
+        localStorage.setItem('home:welcome:show', 'false');
+      });
+      Login.signIn();
+      Loader.loading();
+    });
+
+    afterEach(() => {
+      esApiClient.deleteIndex(oldFormatReportingIndex);
+      esApiClient.deleteDataStream('.kibana-reporting-.kibana_admins_group');
+      kbnApiClient.deleteSampleData('ecommerce', userCredentials);
+      kbnApiAdvancedClient.deleteSavedObjects('admin:dev');
+      esApiAdvancedClient.pruneAllReportingIndices();
+    });
+
+    it('should correctly display all reports for both old reporting index and a new reporting stream data', () => {
+      Home.loadSampleData();
+      Discover.openDataViewPage();
+      Discover.saveReport(newFormatReportingName);
+      Discover.exportToCsv();
+      Reporting.openReportingPage('kibanaNavigation');
+      Reporting.verifySavedReport([newFormatReportingName, oldFormatReportingName]);
+      Reporting.removeReport(newFormatReportingName);
+      Reporting.verifySavedReport([oldFormatReportingName]);
+      Reporting.removeReport(oldFormatReportingName);
+      Reporting.verifySavedReport([]);
+      IndexLifecyclesPolicies.openIndexLifecyclePolicy();
+      IndexLifecyclesPolicies.verifyIndexLifecyclePolicy();
+    });
+  });
+} else {
+  describe.skip('Reporting', () => {
```

Comment:
How it's reported? Because I understand that for versions below 8.15.0 there is no test because it makes no sense. So, I would not say it's skipped but there is no such test in this case.

Reply:
yes, it makes no sense it's reported this way
![image](https://github.com/user-attachments/assets/278070ea-006a-4623-a301-34ef7bb57ffa)